contracts/libraries/PostLib.sol

```diff
@@ -339,32 +342,59 @@ library PostLib  {
     /// @param userAddr Author of the comment
     /// @param postId The post where the comment will be post
     /// @param ipfsHash IPFS hash of document with post information
+    /// @param tags New tags in post (empty array if tags dont change)
+    /// @param communityId New community Id (current community id if dont change)
+    /// @param postType New post type (current community Id if dont change)
     function editPost(
         PostCollection storage self,
         address userAddr,
         uint256 postId,
         bytes32 ipfsHash,
-        uint8[] memory tags
+        uint8[] memory tags,
+        uint32 communityId, 
+        PostType postType
     ) public {
         PostContainer storage postContainer = getPostContainer(self, postId);
-        self.peeranhaCommunity.checkTags(postContainer.info.communityId, tags);
+        if(userAddr == postContainer.info.author) {
```

Comment:
I don't expect these changes after latest changes for edit post merged to develop

Reply:
**Fixed**